common/test/acceptance/pages/studio/utils.py

```diff
@@ -246,11 +246,17 @@ def verify_ordering(test_class, page, expected_orderings):
 
 
 def click_studio_help(page):
-    """Click the Studio help link in the page footer."""
-    page.q(css='.cta-show-sock').click()
+    """
+    Click the Studio help link in the page footer.
+    """
+    help_link_selector = '.cta-show-sock'
+
```

Comment:
I think we can add a comment here about why are we doing this for code understandability.